<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportDialog</class>
 <widget class="QDialog" name="ReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Report</string>
  </property>
  <layout class="QVBoxLayout" name="vbox">
   <item>
    <layout class="QHBoxLayout" name="filters">
     <item>
      <widget class="QDateEdit" name="fromDate">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(222, 222, 222);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="toDate">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(222, 222, 222);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="categoryCombo">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(203, 203, 203);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="queryEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(222, 222, 222);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="placeholderText">
        <string>Search note/merchant</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyBtn">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(203, 203, 203);</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="sp">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(95, 95, 95);
color: rgb(238, 238, 238);</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
